### ISSUE TITLE
How to get this file InceptionResnetV1_epoch001_iter024000_checkpoint.pth ?

### ISSUE THREAD

**Issue by ajithvcoder** [yoyomimi/TNT_pytorch#3]
https://github.com/yoyomimi/TNT_pytorch/blob/master/configs/facenet_triplet_appearance.yaml

RESUME_PATH: 'work_dirs/facenet_triplet_appearance/2020-03-11-05-18/InceptionResnetV1_epoch001_iter024000_checkpoint.pth'

How is above file generated ?

Did you use    20180402-114759-vggface2.pt from https://github.com/timesler/facenet-pytorch/releases as this file InceptionResnetV1_epoch001_iter024000_checkpoint.pth  or else did you do some other thing

**Comment by yoyomimi** [comment]
1. run 'python3 appearance/train_appearance.py --cfg configs/facenet_triplet_appearance.yaml' to get InceptionResnetV1_epoch001_iter024000_checkpoint.pth

2. The .pt file with the same name may work well in this project, and we use 20180402-114759-vggface2.pt as pretrained model for facenet_triplet_appearance training.

**Comment by ajithvcoder** [comment]
When i was trying to train train_appearance.py file it was asking me path for this "InceptionResnetV1_epoch001_iter024000_checkpoint.pth" . Will check it and let you know

**Comment by ajithvcoder** [comment]
HI @yoyomimi  when i run python3 appearance/train_appearance.py  its asking me this file InceptionResnetV1_epoch001_iter024000_checkpoint.pth

**Comment by yoyomimi** [comment]
> HI @yoyomimi when i run python3 appearance/train_appearance.py its asking me this file InceptionResnetV1_epoch001_iter024000_checkpoint.pth

set configs/facenet_triplet_appearance.yaml
MODEL:
  RESUME_PATH: 'work_dirs/facenet_triplet_appearance/2020-03-11-05-18/InceptionResnetV1_epoch001_iter024000_checkpoint.pth'

=>
MODEL:
    RESUME_PATH: ''

and then try again.

**Comment by ajithvcoder** [comment]
@yoyomimi  it  gave error for that "file not found " .also i have another doubt  i am working on this paper "Eye in the Sky: Drone-Based Object Tracking and 3D Localization" by @Haotian-Zhang   in this paper in 4.2 implementations - object detection they have mentioned that we have to run for 20000 epochs . but in your repo 1 epoch takes 2000 iterations (30 minutes in a 12Gb GPU) . So is it enough if i run for 8 epochs(24k iterations) to match that papers result ?

**Comment by yoyomimi** [comment]
> @yoyomimi it gave error for that "file not found " .also i have another doubt i am working on this paper "Eye in the Sky: Drone-Based Object Tracking and 3D Localization" by @Haotian-Zhang in this paper in 4.2 implementations - object detection they have mentioned that we have to run for 20000 epochs . but in your repo 1 epoch takes 2000 iterations (30 minutes in a 12Gb GPU) . So is it enough if i run for 8 epochs(24k iterations) to match that papers result ?

For the first question, you can check the file utils/utils.py line 120, if the resume model path doesn't exist, it will not load the checkpoint file. So I suggested you set MODEL.RESUME_PATH in the config file to set the resume model path to ''. You can check whether the resume model path is '' in the utils/utils.py function load_checkpoint.

For the second question, you can try this first and analyze your loss curve of your training process.

**Comment by ajithvcoder** [comment]
okay thank you @yoyomimi
